crates/floresta-wire/Cargo.toml

```diff
@@ -33,6 +33,10 @@ floresta-compact-filters = { path = "../floresta-compact-filters" }
 thiserror = "1.0"
 floresta-common = { path = "../floresta-common" }
 oneshot = "0.1.5"
+zstd = "0.13.1"
```

Comment:
Does this needs to go in the dev-dependencies ?

Reply:
Yes. We won't need it for anything other than testing IIRC